site-src/geps/gep-1757.md

```diff
@@ -0,0 +1,224 @@
+# GEP-1757: In Cluster Gateway Deployments
```

Comment:
Nit: GEPs should be tied to an issue number so we can track the lifecycle of a GEP after the initial PR is merged.

Reply:
Ah that makes much more sense than making a dummy PR to get a number assigned :upside_down_face: thanks, will change it